qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+Feature: 1. Sign in to Quantori Scoring Portal with Google account feature
+
+ Background:
```

Comment:
Too detailed background

Reply:
What are the details you propose to be deleted?

Reply:
I'd delete all components description on 2 pages, at maximum the only thing you are interested in is for "Sign in with Google" button to be active

Reply:
@shepelevaolga, could you please. comment on that matter?
For me it seems like components description is quite important, so I'd rather keep these details.